Libraries/src/Amazon.Lambda.Serialization.Json/Amazon.Lambda.Serialization.Json.csproj

```diff
@@ -9,15 +9,15 @@
     <AssemblyName>Amazon.Lambda.Serialization.Json</AssemblyName>
     <PackageId>Amazon.Lambda.Serialization.Json</PackageId>
     <PackageTags>AWS;Amazon;Lambda</PackageTags>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>2.0.1</VersionPrefix>
```

Comment:
Newtonsoft.json a transitive dependency for consumers - wont this break binary compatibility?  Thinking we should at least make this a minor bump, though a major bump could also be appropriate.  Thoughts?

Reply:
Lets bump the version to 2.1.0

Reply:
Updated.